instrumentation/internal/internal-reflection/bootstrap/src/main/java/io/opentelemetry/javaagent/instrumentation/internal/reflection/VirtualFieldDetector.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.internal.reflection;
```

Comment:
WDYT about putting this class in `javaagent-bootstrap` instead of a new bootstrap module? It's used by the `javaagent-tooling` module, so it makes sense for it to be in the `javaagent-bootstrap` module.